### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by eri-b**

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = User.find_by(id: params[:session][:user_id])
```
Comment (rvvergara) [inline app/controllers/sessions_controller.rb]: This isn't a good way to log users in. Nobody actually remembers his/her id that's stored in the database :) 

We can change this by:

```ruby
user = User.find_by(name: params[:session][:name]
```

Then we need to make some changes in `views/sessions/new` as well to reflect the change in parameters we are using.

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<h1>Log in</h1>
+
+<%= form_for(:session, url: login_path) do |f| %>
+
+  <%= f.label :user_id %>
```
Comment (rvvergara) [inline app/views/sessions/new.html.erb]: Instead of `user_id` we should be referring to `:name` here in both the label and the input field

**Comment by HenryTabima** [comment]
# Hello!
Great job! :clap: Just a few more things to do here.

To consider the project 'complete' it must satisfy the following points:

#### Behaviour
- [ ] It has functional `login` page (authentication WITHOUT password, just providing a user email)
- [ ] The event's show page displays the list of attendants (there is no link to see them, please add links)
- [ ] The user's show page displays events the user has created and events that the user is attending to separated in past and upcoming

##### UsersController
###### `show` action
- [ ] lists events previoulsy attendented by user **using event model scope**, i.e. `@past_events = current_user.events.previous`
- [ ] lists upcoming events previoulsy attended by user **using event model scope**, i.e. `@upcoming_events = current_user.events.upcoming`

### Crucial things to make project works
- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.

### Personal observations
- I only can log in creating a new account, I should be able to log out and log in again with the same account
- for every view should be a link to it (there are no links to show post, show users, show current user). Remember that this is going to be part of your portfolio and the ones that check this project shouldn't be navigating hard coding the URLs
- the Readme file is fine but I'm sure that you can do it better :100: 

If you have any questions feel free to reach me out on slack
:computer: Happy Coding!

**Comment by azdrenymeri** [comment]
## Hello @DevMFernandes  and @eri-b :wave: 
#### Nice job on implementing the changes 
#### There are no further requests from my side you can go ahead and merge it

**Pull request merged**